### ISSUE TITLE
Importing playlists ignores songs with unicode in the name

### ISSUE THREAD

**Issue by EnderShadow** [quodlibet/quodlibet#3909]
Steps to reproduce
------------------
1. Import a playlist with unicode characters in the song names.

Expected Output
---------------
The playlist should be imported and contain all songs from the imported playlist.

Actual Output
-------------
All songs except for those with unicode in the name were imported.

Test System
-----------

### Which version of Quod Libet?
4.4.0

### Which operating system 
Arch Linux

Additional Information
----------------------
quodlibet spit out these errors on the terminal.

`E: 4.983: formats._misc.MusicFile: _util.py:272:_openfile: quodlibet.formats._misc.AudioFileError: [Errno 2] No such file or directory: '/home/matthew/Media Player/exported/file:/home/matthew/Music/Absolute%20Duo/01.%20Believe×Believe.mp3'`
`E: 4.983: formats._misc.MusicFile: _util.py:272:_openfile: quodlibet.formats._misc.AudioFileError: [Errno 2] No such file or directory: '/home/matthew/Media Player/exported/file:/home/matthew/Music/Absolute%20Duo/03.%202／2.mp3'`

The playlist itself is attached (but renamed as a .txt file) and all files in the playlist do exist at the paths specified.
[Absolute Duo.txt](https://github.com/quodlibet/quodlibet/files/8011195/Absolute.Duo.txt)

**Comment by afontenot** [comment]
> to know if it's my bad for putting utf-8 in a uri or not

Yes, I believe it is. The [spec](https://www.w3.org/Addressing/rfc1738.txt) says:

> URLs are written only with the graphic printable characters of the US-ASCII coded character set.

Non-ASCII characters need to be percent encoded. I'd just use paths instead of a URI to solve this problem.
